### PR TITLE
Add missing container links

### PR DESCRIPTION
**Pull request by negbie**

### CODE REVIEW

File: pages/installation/docker.rst [hunk 1]
```diff
@@ -79,6 +79,8 @@ Example::
       image: docker.elastic.co/elasticsearch/elasticsearch:5.5.1
       environment:
         - http.host=0.0.0.0
+        - transport.host=localhost
+        - network.host=0.0.0.0
```
Comment (joschi) [inline pages/installation/docker.rst:83]: These settings aren't required for the example to work. Why did you add them?

File: pages/installation/docker.rst [hunk 2]
```diff
@@ -96,6 +98,9 @@ Example::
         # Password: admin
         - GRAYLOG_ROOT_PASSWORD_SHA2=8c6976e5b5410415bde908bd4dee15dfb167a9c873fc4bb8a81f6f2ab448a918
         - GRAYLOG_WEB_ENDPOINT_URI=http://127.0.0.1:9000/api
+      links:
+        - mongodb:mongo
+        - elasticsearch
```
Comment (joschi) [inline pages/installation/docker.rst:103]: [`depends_on`](https://docs.docker.com/compose/compose-file/#depends_on) and [`links`](https://docs.docker.com/compose/compose-file/#links) pretty much do the same thing.

The example files are working for me. Do you have specific problems with them?

Also see: https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/

**Review by joschi — CHANGES_REQUESTED** [review]

File: pages/installation/docker.rst (hunk 2 above)
Comment (negbie) [reply pages/installation/docker.rst:103]: I tried your example on a rancher cluster and had two problems. Mongodb and es could not connect. Wish my changes everything came up nicely.

File: pages/installation/docker.rst (hunk 1 above)
Comment (negbie) [reply pages/installation/docker.rst:83]: Without this i've always got "Couldn't read cluster health for indices [graylog_*] (Could not connect to elasticsearch)"

**Comment by negbie** [comment]
Hi Joschi,
i created a new DigitalOcean dropplet with Docker 17.06.1. Took your example and this is what i see in the logs:
```
graylog_1        | 2017-08-22 19:23:49,774 INFO : org.mongodb.driver.cluster - Exception in monitor thread while connecting to server mongo:27017
graylog_1        | com.mongodb.MongoSocketException: mongo: Name or service not known
graylog_1        |      at com.mongodb.ServerAddress.getSocketAddress(ServerAddress.java:188) ~[graylog.jar:?]
graylog_1        |      at com.mongodb.connection.SocketStreamHelper.initialize(SocketStreamHelper.java:57) ~[graylog.jar:?]
graylog_1        |      at com.mongodb.connection.SocketStream.open(SocketStream.java:58) ~[graylog.jar:?]
graylog_1        |      at com.mongodb.connection.InternalStreamConnection.open(InternalStreamConnection.java:115) ~[graylog.jar:?]
graylog_1        |      at com.mongodb.connection.DefaultServerMonitor$ServerMonitorRunnable.run(DefaultServerMonitor.java:113) [graylog.jar:?]
```
Then i added
```
+      links:
+        - mongodb:mongo
+        - elasticsearch
```
Ok now I could establish a connection to mongodb. But Graylog is still not useable because of this:
```
graylog_1        | 2017-08-22 19:30:28,162 ERROR: org.graylog2.indexer.cluster.Cluster - Couldn't read cluster health for indices [graylog_*] (elasticsearch)
graylog_1        | 2017-08-22 19:30:28,162 INFO : org.graylog2.periodical.IndexerClusterCheckerThread - Indexer not fully initialized yet. Skipping periodic cluster check.
```
Inside the Graylog GUI i get the warning:
Could not load field information
Loading field information failed with status: cannot GET http://MASKED:9000/api/system/fields (500)

Then i added
```
+        - transport.host=localhost
+        - network.host=0.0.0.0
```
Everything works.

So it's no a rancher problem. Which docker version did you use?

**Review by joschi — APPROVED** [review]
Fair enough. If everything works with these changes, we'll merge it.

**Pull request merged**